src/test/java/org/scijava/script/ScriptEngineTest.java

```diff
@@ -99,6 +101,11 @@ public ScriptEngine getScriptEngine() {
 		public List<String> getExtensions() {
 			return Arrays.asList("rot13");
 		}
+
+        @Override
+        public AutoCompleter getAutoCompleter(ScriptEngine scriptEngine) {
+            return new DefaultAutoCompleter(this, scriptEngine);
```

Comment:
This override is unnecessary, no? Because `AbstractScriptLanguage` already defines this same logic?